### PR TITLE
[release-4.7.0] upgrade guide and release notes

### PR DESCRIPTION
**Pull request by sijie**
- add upgrade for 4.6.x to 4.7.0
- backfill upgrade guide in old releases
- add release notes for 4.7.0
- add 4.7.0 to releases

### CODE REVIEW

**Comment by sijie** [comment]
NOTE: uncompleted BPs are not highlighted. Those BPs will be deferred to next release.

File: site/docs/4.7.0/overview/releaseNotes.md [hunk 1]
```diff
@@ -2,16 +2,106 @@
...
+#### CRC32C
+
+`circe-checksum` module is ported from Apache Pulsar to Apache BookKeeper, and CRC32C digest type is added as one digest type option.
+The JNI based CRC32C in `circe-checksum` module provides excellent performance than existing CRC32 digest type. Users are encouraged
+to start use CRC32C digest type.
+
+### Security
+
+- New PEM format `X.509` certificates are introduced for TLS authentication. See [#965](https://github.com/apache/bookkeeper/pull/965) for more details.
+- TLS related settings are converaged into same settings as bookie server. See [Upgrade Guide](../../admin/upgrade) for more details.
```
Comment (eolivelli) [inline site/docs/4.7.0/overview/releaseNotes.md]: typo: converaged

File: site/docs/4.7.0/admin/upgrade.md [hunk 2]
```diff
@@ -59,6 +59,105 @@ For each Bookie:
...
+
+#### Client Configuration Changes
+
+##### New Settings
+
+Following settings are newly added in 4.7.0.
+
+| Name | Default Value | Description |
+|------|---------------|-------------|
+| maxNumEnsembleChanges | Integer.MAX\_VALUE | The max allowed ensemble change number before sealing a ledger on failures |
```
Comment (jiazhai) [inline site/docs/4.7.0/admin/upgrade.md]: `Integer.MAX\_VALUE` should be `Integer.MAX_VALUE`?

File: site/docs/latest/admin/upgrade.md [hunk 3]
```diff
@@ -59,6 +59,106 @@ For each Bookie:
...
+
+#### Client Configuration Changes
+
+##### New Settings
+
+Following settings are newly added in 4.7.0.
+
+| Name | Default Value | Description |
+|------|---------------|-------------|
+| maxNumEnsembleChanges | Integer.MAX\_VALUE | The max allowed ensemble change number before sealing a ledger on failures |
```
Comment (jiazhai) [inline site/docs/latest/admin/upgrade.md]: so is here?

File: site/docs/4.7.0/admin/upgrade.md (hunk 2 above)
Comment (sijie) [reply site/docs/4.7.0/admin/upgrade.md]: \ is to escape `_`

File: site/docs/latest/admin/upgrade.md (hunk 3 above)
Comment (sijie) [reply site/docs/latest/admin/upgrade.md]: \ is to escape `_`

File: site/docs/4.7.0/overview/releaseNotes.md (hunk 1 above)
Comment (sijie) [reply site/docs/4.7.0/overview/releaseNotes.md]: fixed

**Pull request closed without merging**